cuebot/src/main/java/com/imageworks/spcue/service/JobManagerSupport.java

```
@@ -150,6 +154,17 @@ public boolean shutdownJob(JobInterface job, Source source, boolean isManualKill
                  */
                 emailSupport.sendShutdownEmail(job);
 
+                // Record job completion metric
+                if (prometheusMetrics != null && showDao != null) {
+                    try {
+                        String showName = showDao.getShowDetail(job.getShowId()).getName();
+                        String state = isManualKill ? "KILLED" : "FINISHED";
+                        prometheusMetrics.recordJobCompleted(state, showName);
+                    } catch (Exception e) {
```

Don't swallow exception

---

Done. See commit: https://github.com/AcademySoftwareFoundation/OpenCue/pull/2086/commits/3c0686e8a6179163aeda1fe49bf5a377e7232651